.github/workflows/container-image.yml

```
@@ -53,6 +53,12 @@ jobs:
         if: ${{ startsWith(github.ref, 'refs/tags/') }}
         run: |
           echo TAG_NAME=$(echo $GITHUB_REF | sed -e "s|refs/tags/||") >> $GITHUB_ENV
```

What about reusing this step and do something like:

```suggestion
          echo TAG_NAME=$(echo $GITHUB_REF | sed -e "s|refs/tags/||") >> $GITHUB_ENV
          echo REPOSITORY_NAME=$(echo {{ github.repository_owner }} | awk "{print tolower($0)}")) >> $GITHUB_ENV
````

I didn't try it on GH, but I think it should work and would just reduce the number of steps and keep things consistent

---

> BTW, I'm sorry I didn't see that coming with my tests :(

No worries, you tested everything else on your fork 😊.